packages/synapse-react-client/src/synapse-queries/entity/useExportToCavatica.ts

```diff
@@ -14,10 +14,17 @@ export function addDrsUriToSql(
   fileIdColumnName?: string,
   fileNameColumnName?: string,
   fileVersionColumnName?: string,
+  removeSpacesFromSelectColumns: boolean = false,
 ): string {
   const selectColumnsList = selectColumns
     ?.filter(col => col.name != 'name')
-    .map(col => `"${col.name}"`)
+    .map(col => {
+      if (removeSpacesFromSelectColumns) {
+        return `"${col.name}" as "${col.name.replace(/\s/g, '')}"`
```

Comment:
The bug fix - alias select columns and remove whitespace.
(removeSpacesFromSelectColumns set to true for pfb export)

Reply:
Couple of edge cases I verified:

- if there is a WHERE clause in the originalSql, it still works (e.g. `SELECT "my col" as "mycol" from syn123 WHERE "my col" = 'foo'` correctly applies a filter)
- there is no need to escape `"` (Synapse does not seem to allow it in a valid column name)

so this change LGTM